octobot_trading/exchanges/traders/trader.py

```diff
@@ -385,6 +401,36 @@ async def chain_order(self, order, chained_order, update_with_triggering_order_f
         order.add_chained_order(chained_order)
         self.logger.info(f"Added chained order [{chained_order}] to [{order}] order.")
 
+    async def update_order_as_inactive(
+        self, order, ignored_order=None, wait_for_cancelling=True,
+        cancelling_timeout=octobot_trading.constants.INDIVIDUAL_ORDER_SYNC_TIMEOUT
+    ) -> bool:
+        if self.simulate:
+            self.logger.error(f"Can't update order as inactive on simulated trading.")
+            return False
```

Comment:
Shouldn't we check ` self.enable_inactive_orders` instead `self.simulate` here ?

Reply:
indeed!